README.md

```diff
@@ -1 +1,25 @@
-# status-network
\ No newline at end of file
+# status-network
+
+### Testing locally
+
+Make sure you have node.js installed first.
+
+1. Open Terminal and navigate to the project root directory,
+
+2. Run `yarn install`,
+
+3. We share common elements across our sites using submodules, so you'll need to get those too:
+
+    ```bash
+    git submodule update --init --recursive
+    cd themes/navy/layout/partial/shared-partials
```

Comment:
I don't think this is used anymore...

I don't have `themes/navy/layout/partial/shared-partials` and building the site works fine. We should remove `.gitmodules` and other references to submodules.

Reply:
@jakubgs Sorry, forgot to remove the No.3. I've removed these unnecessary lines.